examples/reference/ambientLight/ambientLight1/ambientLight1.rpde

```diff
@@ -0,0 +1,18 @@
+P3D <- "processing.opengl.PGraphics3D"
+PI <- pi
```

Comment:
Good -- `PI <- pi` is better than `PI <- 3.141592` -- but still a placeholder until PConstants are defined and loaded outside the example sketches.

In the meantime perhaps all examples using `PI`, `HALF_PI`, `TWO_PI` etc. should be updated to be defined in terms of the R built-in `pi`.